app/assets/stylesheets/modules/media_queries.css.scss

```diff
@@ -19,7 +19,7 @@
 	}
 }
 
-@media (max-width: 1349px) {
+@media (max-width: 1549px) {
```

Comment:
This approach seems fragile and likely to break again. Can you find something more robust?

Maybe use flexbox with https://github.com/10up/flexibility ?